airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -373,6 +378,19 @@ def __init__(
         self.deferrable = deferrable
         self.poll_interval = poll_interval
         self.remote_pod: k8s.V1Pod | None = None
+        if is_delete_operator_pod is not None:
+            warnings.warn(
+                "`is_delete_operator_pod` parameter is deprecated, please use `on_finish_action`",
+                DeprecationWarning,
```

Comment:
```suggestion
                DeprecationWarning,
                stacklevel=2,
```

We probably should do this (think this is the right level).